src/App.tsx

```diff
@@ -15,71 +16,88 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
+  state = {
+    selectedGood: 'Jam is selected',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  handleClickAdd = (item: string) => (
+    this.state.selectedGood === `${item} is selected`
+      ? this.setState({ selectedGood: 'No goods selected' })
+      : this.setState({ selectedGood: `${item} is selected` })
```

Comment:
looks kinda hardcoded. Checking for exact match with a strings is not the best option. You'll handle it better if you fix my previous comment